definitions/scenarios/content.rb

```diff
@@ -8,9 +8,18 @@ class Prepare < ForemanMaintain::Scenario
       end
 
       def compose
+        current_downstream_feature = feature(:instance).downstream
         # FIXME: remove this condition on next downstream upgrade scenario
         if Procedures::Content::Prepare.present?
+          if current_downstream_feature && current_downstream_feature.current_minor_version == '6.9'
```

Comment:
would this only be run on 6.9 or is there potential that this could be run later? i.e. should we allow users to run after 6.9 as well?

Reply:
It's only on 6.9 that we want to do the enabling/disabling and starting/stopping of the pulpcore services.  Otherwise we can just run the migration.